### PR TITLE
PHPCS fixes for rtmedia core

### PR DESCRIPTION
**Pull request by vaishaliagola27**
Related Issue #1147

### CODE REVIEW

File: app/main/controllers/shortcodes/RTMediaGalleryShortcode.php [hunk 1]
```diff
@@ -107,7 +107,7 @@ public static function register_scripts() {
 				'rtmedia_plupload_files_filter',
 				array(
 					array(
-						'title'      => 'Media Files',
+						'title'      => __( 'Media Files', 'buddypress-media' ),
```
Comment (thrijith) [inline app/main/controllers/shortcodes/RTMediaGalleryShortcode.php]:
```suggestion
						'title'      => esc_html__( 'Media Files', 'buddypress-media' ),
```

File: app/main/controllers/template/rtmedia-actions.php [hunk 2]
```diff
@@ -368,13 +366,10 @@ function rtmedia_link_in_footer() {
...
 
 	if ( $link ) {
-
-		$aff_id = ( '' !== $option['rtmedia_affiliate_id'] ) ? '&ref=' . $option['rtmedia_affiliate_id'] : '';
-		$href   = 'https://rtmedia.io/?utm_source=dashboard&utm_medium=plugin&utm_campaign=buddypress-media' . $aff_id;
 		?>
 		<div class='rtmedia-footer-link'>
 			<?php esc_html_e( 'Empowering your community with ', 'buddypress-media' ); ?>
-			<a href='<?php echo esc_url( $href ); ?>' title='<?php esc_attr_e( 'The only complete media solution for WordPress, BuddyPress and bbPress', 'buddypress-media' ); ?> '>rtMedia</a>
+			<a href='<?php echo esc_url( 'https://rtmedia.io/?utm_source=dashboard&utm_medium=plugin&utm_campaign=buddypress-media' ); ?>' title='<?php esc_attr_e( 'The only complete media solution for WordPress, BuddyPress and bbPress', 'buddypress-media' ); ?> '>rtMedia</a>
```
Comment (thrijith) [inline app/main/controllers/template/rtmedia-actions.php]: Don't need escaping for static URL.

File: app/main/controllers/template/RTMediaNav.php [hunk 3]
```diff
@@ -284,9 +284,7 @@ public function sub_nav() {
 
 		$albums = '';
 		// condition to keep "Album" tab active.
-		if ( array_key_exists( 'media_type', $rtmedia_query->query ) && isset( $rtmedia_query->query['media_type'] ) && ( 'album' === $rtmedia_query->query['media_type'] ) ) {
-			$albums = 'current selected';
-		} elseif ( array_key_exists( 'media_type', $rtmedia_query->action_query ) && isset( $rtmedia_query->action_query->media_type ) && ( 'album' === $rtmedia_query->action_query->media_type ) ) {
+		if ( ( array_key_exists( 'media_type', $rtmedia_query->query ) && isset( $rtmedia_query->query['media_type'] ) && ( 'album' === $rtmedia_query->query['media_type'] ) ) || ( array_key_exists( 'media_type', $rtmedia_query->action_query ) && isset( $rtmedia_query->action_query->media_type ) && ( 'album' === $rtmedia_query->action_query->media_type ) ) ) {
```
Comment (thrijith) [inline app/main/controllers/template/RTMediaNav.php]: Could we split into multiple lines for better readability?

File: app/main/controllers/template/rtmedia-filters.php [hunk 4]
```diff
@@ -131,9 +131,7 @@ function rtm_is_buddypress_enable( $flag ) {
 
 	global $rtmedia_query;
 
-	if ( isset( $rtmedia_query->query ) && isset( $rtmedia_query->query['context'] ) && 'group' === $rtmedia_query->query['context'] && is_rtmedia_group_media_enable() ) {
-		return $flag;
-	} elseif ( isset( $rtmedia_query->query ) && isset( $rtmedia_query->query['context'] ) && 'profile' === $rtmedia_query->query['context'] && is_rtmedia_profile_media_enable() ) {
+	if ( ( isset( $rtmedia_query->query ) && isset( $rtmedia_query->query['context'] ) && 'group' === $rtmedia_query->query['context'] && is_rtmedia_group_media_enable() ) || ( isset( $rtmedia_query->query ) && isset( $rtmedia_query->query['context'] ) && 'profile' === $rtmedia_query->query['context'] && is_rtmedia_profile_media_enable() ) ) {
```
Comment (thrijith) [inline app/main/controllers/template/rtmedia-filters.php]: Could we split into multiple lines for better readability?

File: app/main/controllers/upload/processors/RTMediaUploadFile.php [hunk 5]
```diff
@@ -243,7 +243,7 @@ public function is_valid_type( $file ) {
 				'rtmedia_plupload_files_filter',
 				array(
 					array(
-						'title'      => 'Media Files',
+						'title'      => __( 'Media Files', 'buddypress-media' ),
```
Comment (thrijith) [inline app/main/controllers/upload/processors/RTMediaUploadFile.php]:
```suggestion
						'title'      => esc_html__( 'Media Files', 'buddypress-media' ),
```

File: app/main/controllers/template/RTMediaNav.php (hunk 3 above)
Comment (vaishaliagola27) [reply app/main/controllers/template/RTMediaNav.php]: okay

File: app/main/controllers/privacy/RTMediaPrivacy.php [hunk 6]
```diff
@@ -466,7 +466,7 @@ public function rt_privacy_settings_action() {
 		$default_privacy = sanitize_text_field( filter_input( INPUT_POST, 'rtmedia-default-privacy', FILTER_SANITIZE_STRING ) );
 		$nonce           = sanitize_text_field( filter_input( INPUT_POST, 'rtmedia_member_settings_privacy', FILTER_SANITIZE_STRING ) );
 
-		if ( ! empty( $default_privacy ) ) {
+		if ( ! empty( $default_privacy ) || '0' === $default_privacy ) {
```
Comment (thrijith) [inline app/main/controllers/privacy/RTMediaPrivacy.php]: Could  we convert `$default_privacy` to `int` and then compare?

File: app/main/controllers/template/rtmedia-actions.php [hunk 7]
```diff
@@ -368,13 +366,10 @@ function rtmedia_link_in_footer() {
...
 
 	if ( $link ) {
-
-		$aff_id = ( '' !== $option['rtmedia_affiliate_id'] ) ? '&ref=' . $option['rtmedia_affiliate_id'] : '';
-		$href   = 'https://rtmedia.io/?utm_source=dashboard&utm_medium=plugin&utm_campaign=buddypress-media' . $aff_id;
 		?>
 		<div class='rtmedia-footer-link'>
 			<?php esc_html_e( 'Empowering your community with ', 'buddypress-media' ); ?>
-			<a href='<?php echo esc_url( $href ); ?>' title='<?php esc_attr_e( 'The only complete media solution for WordPress, BuddyPress and bbPress', 'buddypress-media' ); ?> '>rtMedia</a>
+			<a href='<?php echo 'https://rtmedia.io/?utm_source=dashboard&utm_medium=plugin&utm_campaign=buddypress-media'; ?>' title='<?php esc_attr_e( 'The only complete media solution for WordPress, BuddyPress and bbPress', 'buddypress-media' ); ?> '>rtMedia</a>
```
Comment (thrijith) [inline app/main/controllers/template/rtmedia-actions.php]: We probably don't need `echo` here for static value.

File: app/main/controllers/privacy/RTMediaPrivacy.php (hunk 6 above)
Comment (vaishaliagola27) [reply app/main/controllers/privacy/RTMediaPrivacy.php]: It is stored as string and used at that everywhere. So i think it's fine here.

Comment (thrijith) [reply app/main/controllers/privacy/RTMediaPrivacy.php]: Ok, I meant something similar to what is done here https://github.com/rtMediaWP/rtMedia/pull/1538/files#diff-3968235ba0fab09fdf96bb65f5a42a77R517

Comment (vaishaliagola27) [reply app/main/controllers/privacy/RTMediaPrivacy.php]: oh okay. Changing it.

**Pull request merged**